android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -284,25 +363,54 @@ public void onError(String info) {
         if (majorProtocolVersion >= 5 && !isCapabilitySupported) {
             DebugTool.logError(TAG, "Video streaming not supported on this module");
             stateMachine.transitionToState(StreamingStateMachine.ERROR);
             return;
         }
-        // regardless of VideoStreamingParameters are specified or not, we should refer to VideoStreamingCapability.
+        processCapabilitiesWithPendingStart(encrypted, parameters);
+    }
+
+    private void configureGlobalParameters(Context context, Class<? extends SdlRemoteDisplay> remoteDisplayClass, boolean encrypted, VideoStreamingRange portraitRange, VideoStreamingRange landscapeRange) {
+        this.context = new WeakReference<>(context);
+        this.remoteDisplayClass = remoteDisplayClass;
+        this.isEncrypted = encrypted;
+        this.majorProtocolVersion = internalInterface.getProtocolVersion().getMajor();
+        this.supportedPortraitStreamingRange = portraitRange;
+        this.supportedLandscapeStreamingRange = landscapeRange;
+    }
+
+
+    private void processCapabilitiesWithPendingStart(boolean encrypted, VideoStreamingParameters parameters) {
+        final VideoStreamingParameters params = (parameters == null) ? new VideoStreamingParameters() : new VideoStreamingParameters(parameters);
         if (majorProtocolVersion >= 5) {
             if (internalInterface.getSystemCapabilityManager() != null) {
-                final VideoStreamingParameters params = ( parameters == null) ? new VideoStreamingParameters() : new VideoStreamingParameters(parameters);
                 internalInterface.getSystemCapabilityManager().getCapability(SystemCapabilityType.VIDEO_STREAMING, new OnSystemCapabilityListener() {
                     @Override
                     public void onCapabilityRetrieved(Object capability) {
-                        params.update((VideoStreamingCapability) capability, vehicleMake);    //Streaming parameters are ready time to stream
-                        startStreaming(params, encrypted);
+                        VideoStreamingCapability castedCapability = ((VideoStreamingCapability) capability);
+                        VideoStreamManager.this.originalCapability = castedCapability;
+
+                        params.update(castedCapability, vehicleMake);    //Streaming parameters are ready time to stream
+                        VideoStreamManager.this.parameters = params;
+
+                        VideoStreamingCapability capabilityToSend = new VideoStreamingCapability();
+                        capabilityToSend.setAdditionalVideoStreamingCapabilities(getSupportedCapabilities(castedCapability));
+
+                        if (capabilityToSend.getAdditionalVideoStreamingCapabilities() == null || capabilityToSend.getAdditionalVideoStreamingCapabilities().isEmpty()) {
+                            stateMachine.transitionToState(StreamingStateMachine.STOPPED);
```

Comment:
Can we add a log message here so the developer is aware of why the video has not streamed
```suggestion
                            stateMachine.transitionToState(StreamingStateMachine.STOPPED);
                            DebugTool.logError(TAG, "The Video stream was not started because there were no supported video streaming capabilities, please double check that the VideoStreamRanges provided are not disabled ranges");
```